README.adoc

```diff
@@ -104,3 +104,116 @@ include::topics/templates/document-attributes.adoc[]
 . Open the generated html file to ensure everything looks correct.
 
 You have now created the scaffolding for a new guide. When you are ready for some initial review and feedback, file a pull request to `upstream/master`.
+
+### Merging Contributions
+// title OK?
+// following content up for revision.
+// add note on branching according to issue numbers?
+// externalize naming examples to variables?
+
+. Once the changes in a Pull Request have been approved, you can merge them into `upstream/master`.
++
+[IMPORTANT]
+--
+Do not merge your own Pull Requests unless absolutely necessary. You are accountable for any errors, merge conflicts or bugs resulting from incorrect merging. Let another contributor with the appropriate repository access privileges do it.
+--
+
+. Before merging, verify that you have the Contributor's fork of the link:https://github.com/openshiftio/appdev-documentation[upstream repository] repository in your list of remotes.
++
+[source,options="nowrap"]
+--
+git remote -v
+--
++
+. To add a remote fork for to you Git directory:
++
+.. Navigate to the contributor's fork on link:https://github.com/[Git Hub].
+.. Click the _Clone or Download_ button.
+.. Select _Use SSH_.
+.. Copy the repository URL beginning with `git@github.com:`.
+.. Execute the following command. Replace `username` with the user name of the contributor.
++
+[source,options="nowrap"]
+--
+git remote add username git@github.com:username/appdev-documentation.git
+--
++
+.. Use `git remote -v` to verify that the fork is now listed among your remotes.
+.. Execute `git remote update` to fetch the latest state of the remotes to your working directory.
++
+. Create a new branch and set it to track the remote branch that introduces the changes in the Pull Request.
++
+[source,options="nowrap"]
+--
+git checkout -b pr-to-merge username/featurebranch
+--
++
+----
+Branch pr-to-merge set up to track remote branch <topic_branch> from <username> by rebasing.
+Switched to a new branch 'pr-000'
+----
++
+. Fetch the latest state of `upstream` and rebase your topic branch.
++
+[source,options="nowrap"]
+--
+git fetch upstream
+--
++
+[source,options="nowrap"]
+--
+git rebase upstream/master
+--
++
+If you have multiple commits on your feature branch, squash them before merging.
++
+. To test if all the books build correctly with your changes, execute the script `/scripts/buildGuides.sh`. If the output contains no errors, the guides build correctly.
++
+. Check out your local `master`:
```

Comment:
I'd advocate for using _checkout_ as a verb since it's consistent with official Git documentation and manpages.